sdk/communication/azure-communication-identity/src/main/resources/azure-communication-identity.properties

```diff
@@ -0,0 +1,2 @@
+name=${project.artifactId}
```

Comment:
Looks like an IDE file. Common doesn't have it.

Reply:
This one is actually required for getting the correct user agent in live tests. Common does not have live tests which is why it doesnt have it, all the other SDKs (chat, sms, administration, etc) all have this file.